statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -644,22 +956,38 @@ def bic(self):
 
 
     @cache_readonly
-    def cov_params_default(self):
+    def _cov_params_default(self):
```

Comment:
This is redundant now, isn't it?
(It's not a problem to keep it even if it's not used internally.)